fastlane/metadata/android/en-US/full_description.txt

```diff
@@ -0,0 +1,11 @@
+Unlock the power of AI with Private AI Chat, the ultimate app for secure and private conversations with LLMs. The app brings the sophistication of cutting-edge AI directly to your device, ensuring your chats remain confidential and offline.
+
+Key Features:
+- Offline Chatting: Interact with advanced AI models directly on your device without the need for an internet connection.
+- Supports H2O-Danube & GGUF Formats: At the moment we are supporting H2O-Danube and GGUF.
```

Comment:
The current text "Supports H2O-Danube & GGUF Formats: At the moment we are supporting H2O-Danube and GGUF." is outdated. It reminds me that the app description on the Play Store also needs an update.

We can say something a long the lines of "The app supports GGUF formats, allowing you to either load models locally or search directly in Hugging Face."

Reply:
I indeed took the description from Play Store.